tests/functional/TestSmokeTest/tests/heketi_test.go

```diff
@@ -171,48 +170,33 @@ func HeketiCreateVolumeWithGid(t *testing.T) {
 	volReq.Snapshot.Enable = true
 	volReq.Snapshot.Factor = 1.5
 
-	// Set to the vagrant gid
-	volReq.Gid = 1000
+	volReq.Gid = 2333
 
 	// Create the volume
 	volInfo, err := heketi.VolumeCreate(volReq)
 	tests.Assert(t, err == nil, "expected err == nil, got:", err)
 
-	// SSH into system and create two writers belonging to writegroup gid
-	vagrantexec := na.Use(logger)
+	// SSH into system, create two writers belonging to writegroup gid and
+	// make sure both can write and the sticky group bit is set
+	exec := na.Use(logger)
 	cmd := []string{
-		"sudo groupadd writegroup",
-		"sudo useradd writer1 -G writegroup -p'$6$WBG5yf03$3DvyE41cicXEZDW.HDeJg3S4oEoELqKWoS/n6l28vorNxhIlcBe2SLQFDhqq6.Pq'",
-		"sudo useradd writer2 -G writegroup -p'$6$WBG5yf03$3DvyE41cicXEZDW.HDeJg3S4oEoELqKWoS/n6l28vorNxhIlcBe2SLQFDhqq6.Pq'",
+		"sudo groupadd -f -g 2333 writegroup",
```

Comment:
Do you want to make this conditional like the lines below?

Reply:
no need, because of the `-f`